website/client/components/inventory/stable/petItem.vue

```diff
@@ -28,10 +28,18 @@ div
     left: calc((100% - 62px) / 2);
   }
 
+  .pet-slot {
+    margin: 0 2px;
+  }
+
   .pet-progress-bar {
     height: 4px;
     background-color: #24cc8f;
   }
+
```

Comment:
could you please remove the duplicate css rule?

Reply:
duplicate css rule? @negue

Reply:
nvm I took care of it ;) @negue